generator/src/templates/assembly-package-new.mst.json

```diff
@@ -0,0 +1,166 @@
+{
```

Comment:
I tried going that way, since the manipulation of the theia/package.json requires much more complicated manipulation that is not needed in the linkless case, I decided it would be simplest to introduce new templates.

Reply:
I don't think we need to generate the assembly as part of the build process at all, t.b.h. But I decided leave that for a later PR to keep this smaller.

Reply:
we should not introduce complete new files, we should reuse at maxium the templates.

If assembly folder needs to be committed one day, then generated files would just have to be committed, but during that time we don't have several builds in parallel

It was the goal of using templates and not using plain files (where we do copy of copy for every usecase)